consensus/polybft/bridge/bridge_event_relayer.go

```diff
@@ -136,15 +136,28 @@ func (ber *bridgeEventRelayerImpl) sendSignedBridgeMessageBatch(event *contracts
 		exists             bool
 	)
 
-	if event.DestinationChainID.Cmp(ber.internalChainID) != 0 {
-		txRelayer, exists = ber.externalTxRelayers[destinationChainID]
+	if event.DestinationChainID.Cmp(ber.internalChainID) != 0 && !event.IsRollback ||
```

Comment:
This condition is too complicated. Please expand the `SignedBridgeMessageBatch` struct with functions like:
 - `IsRollbackFromInternalChain`
 - `IsRollbackFromExternalChain`
 
Something more descriptive that will make this condition more simple. THere are too many if/else's here.